Njord/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -83,8 +83,17 @@ private void configureBindings() {
     //CODriver.a().whileTrue(new ClimbCommand(mClimber, 0.7));
     
     Driver.rightBumper().onTrue(new IntakeCommand(m_intakeSubsytem));
+    
+    CoDriver.a().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdQuasistatic(SysIdRoutine.Direction.kForward));
+    CoDriver.b().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdQuasistatic(SysIdRoutine.Direction.kReverse));
+    CoDriver.x().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdDynamic(SysIdRoutine.Direction.kForward));
+    CoDriver.y().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdDynamic(SysIdRoutine.Direction.kReverse));
```

Comment:
`SysIdRoutine.Direction` is imported at line 22. Remove `SysIdRoutine`. 

```suggestion
    CoDriver.a().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdQuasistatic(Direction.kForward));
    CoDriver.b().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdQuasistatic(Direction.kReverse));
    CoDriver.x().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdDynamic(Direction.kForward));
    CoDriver.y().and(CoDriver.rightBumper()).whileTrue(mElevator.sysIdDynamic(Direction.kReverse));
```